src/Directory.Build.props

```diff
@@ -25,7 +25,7 @@
   <ItemGroup>
     <PackageReference Include="StyleCop.Analyzers" Version="1.1.118" PrivateAssets="all" />
     <PackageReference Include="Microsoft.CodeAnalysis" Version="4.2.0" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="7.0.3" >
+    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="7.0.4" >
```

Comment:
>"7.0.4"  [](http://example.com/codeflow?start=76&length=8)

Let's do this in a separate PR. 
Too often, changing analyzers causes many unintended ripples.

Reply:
Not this one, but I'll remove this if you prefer